project-backend/Controllers/WorkerController.cs

```diff
@@ -40,9 +40,9 @@ public IActionResult ApplyForWorker()
             {
                 return Conflict(new Error(ex.Message));
             }
-            catch (UserIsAlreadyWorker ex)
+            catch (UserIsAlreadyWorker)
             {
-                return Forbid(ex.Message);
+                return Forbid();
```

Comment:
Returning a message required some additional configuration. At least it threw some error from regarding a missing Authorization handler. 403-s are pretty intuitive so I think it's fine to just return no message.

Reply:
That's all right indeed, no need for a message.